python/gps/algorithm/config.py

```diff
@@ -19,6 +19,12 @@
                               # objects for each condition.
     # Trajectory optimization.
     'traj_opt': TrajOptLQRPython({}),
+    # Use maximum entropy term in trajectory optimization.
+    'max_ent_traj': 1.0
```

Comment:
Let's default this to 0.0, but set it to 1.0 in the hyperparams for the cost learning experiment.